### PR TITLE
*: update slow log doc

### PR DESCRIPTION
**Pull request by crazycs520**
### What is changed, added or deleted? 

* update slow log doc.

I will update v3.0 after got LGTM

### Which version does your change affect? 
dev, v3.0

### CODE REVIEW

File: dev/how-to/maintain/identify-slow-queries.md [hunk 1]
```diff
@@ -39,13 +42,26 @@ Slow Query 基础信息：
...
+* `Parse_time`：表示这个语句在 parse 阶段（解析 AST 语法树）花费的时间。
+* `Compile_time`：表示这个语句在 compile 阶段（生成执行计划）花费的时间。
 * `Query`：表示 SQL 语句。慢日志里面不会打印 `Query`，但映射到内存表后，对应的字段叫 `Query`。
 * `Digest`：表示 SQL 语句的指纹。
 * `Txn_start_ts`：表示事务的开始时间戳，也是事务的唯一 ID，可以用这个值在 TiDB 日志中查找事务相关的其他日志。
 * `Is_internal`：表示是否为 TiDB 内部的 SQL 语句。`true` 表示 TiDB 系统内部执行的 SQL 语句，`false` 表示用户执行的 SQL 语句。
 * `Index_ids`：表示语句涉及到的索引的 ID。
 * `Succ`：表示语句是否执行成功。
 * `Backoff_time`：表示语句遇到需要重试的错误时在重试前等待的时间，常见的需要重试的错误有以下几种：遇到了 lock、Region 分裂、`tikv server is busy`。
+* `Plan`：表示语句的执行计划，用 select tidb_decode_plan('xxx...') 即可查看该语句的执行计划。
```
Comment (TomShawn) [inline dev/how-to/maintain/identify-slow-queries.md]:
```suggestion
* `Plan`：表示语句的执行计划，用 `select tidb_decode_plan('xxx...')` 命令即可查看该语句的执行计划。
```

File: dev/how-to/maintain/identify-slow-queries.md [hunk 2]
```diff
@@ -39,13 +42,26 @@ Slow Query 基础信息：
...
+
+和事务执行相关的字段：
+
+* `Prewrite_time`：表示事务提交时 prewrite 阶段的耗时。
+* `Commit_time`：表示事务提交时 commit 阶段的耗时。
+* `Get_commit_ts_time`：表示事务提交时获取 commit tso 的耗时。
+* `Local_latch_wait_time`：表示事务在提交前在本地等待 latch 的耗时。
+* `Write_keys`：表示事务提交时写 key 的数量。
+* `Write_size`：表示事务提交时写 key/value 的总大小。
```
Comment (TomShawn) [inline dev/how-to/maintain/identify-slow-queries.md]:
```suggestion
* `Write_size`：表示事务提交时写 key 或 value 的总大小。
```

File: dev/how-to/maintain/identify-slow-queries.md [hunk 3]
```diff
@@ -39,13 +42,26 @@ Slow Query 基础信息：
...
+* `Prewrite_region`：表示事务提交时 prewrite 阶段需要给 TiKV 发送的 RPC 请求数量。
```
Comment (TomShawn) [inline dev/how-to/maintain/identify-slow-queries.md]:
```suggestion
* `Prewrite_region`：表示事务提交时在 prewrite 阶段需要给 TiKV 发送的 RPC 请求数量。
```

File: dev/how-to/maintain/identify-slow-queries.md [hunk 4]
```diff
@@ -39,13 +42,26 @@ Slow Query 基础信息：
...
 * `Succ`：表示语句是否执行成功。
 * `Backoff_time`：表示语句遇到需要重试的错误时在重试前等待的时间，常见的需要重试的错误有以下几种：遇到了 lock、Region 分裂、`tikv server is busy`。
+* `Plan`：表示语句的执行计划，用 select tidb_decode_plan('xxx...') 即可查看该语句的执行计划。
+
+和事务执行相关的字段：
+
+* `Prewrite_time`：表示事务提交时 prewrite 阶段的耗时。
+* `Commit_time`：表示事务提交时 commit 阶段的耗时。
+* `Get_commit_ts_time`：表示事务提交时获取 commit tso 的耗时。
+* `Local_latch_wait_time`：表示事务在提交前在本地等待 latch 的耗时。
```
Comment (TomShawn) [inline dev/how-to/maintain/identify-slow-queries.md]:
```suggestion
* `Local_latch_wait_time`：表示事务提交前在本地等待 latch 的耗时。
```

File: dev/how-to/maintain/identify-slow-queries.md [hunk 5]
```diff
@@ -39,13 +42,26 @@ Slow Query 基础信息：
 
 * `Time`：表示日志打印时间。
 * `Query_time`：表示执行这个语句花费的时间。
+* `Parse_time`：表示这个语句在 parse 阶段（解析 AST 语法树）花费的时间。
```
Comment (shenli) [inline dev/how-to/maintain/identify-slow-queries.md]: parse 阶段（解析 AST 语法树）-> 语法解析阶段

File: dev/how-to/maintain/identify-slow-queries.md [hunk 6]
```diff
@@ -39,13 +42,26 @@ Slow Query 基础信息：
...
+* `Compile_time`：表示这个语句在 compile 阶段（生成执行计划）花费的时间。
```
Comment (shenli) [inline dev/how-to/maintain/identify-slow-queries.md]: compile 阶段（生成执行计划）-> 查询优化阶段

File: dev/how-to/maintain/identify-slow-queries.md (hunk 1 above)
Comment (zz-jason) [inline dev/how-to/maintain/identify-slow-queries.md]:
```suggestion
* `Plan`：表示该语句的对应的执行计划。输出格式为 "select tidb_decode_plan('encoded-execution-plan')"，可通过该 SQL 语句解析出具体的执行计划。
```

File: dev/how-to/maintain/identify-slow-queries.md [hunk 7]
```diff
@@ -39,13 +42,26 @@ Slow Query 基础信息：
...
 * `Txn_start_ts`：表示事务的开始时间戳，也是事务的唯一 ID，可以用这个值在 TiDB 日志中查找事务相关的其他日志。
 * `Is_internal`：表示是否为 TiDB 内部的 SQL 语句。`true` 表示 TiDB 系统内部执行的 SQL 语句，`false` 表示用户执行的 SQL 语句。
 * `Index_ids`：表示语句涉及到的索引的 ID。
 * `Succ`：表示语句是否执行成功。
 * `Backoff_time`：表示语句遇到需要重试的错误时在重试前等待的时间，常见的需要重试的错误有以下几种：遇到了 lock、Region 分裂、`tikv server is busy`。
+* `Plan`：表示语句的执行计划，用 select tidb_decode_plan('xxx...') 即可查看该语句的执行计划。
+
+和事务执行相关的字段：
+
+* `Prewrite_time`：表示事务提交时 prewrite 阶段的耗时。
```
Comment (zz-jason) [inline dev/how-to/maintain/identify-slow-queries.md]:
```suggestion
* `Prewrite_time`：表示事务两阶段提交中第一阶段（prewrite 阶段）的耗时。
```

File: dev/how-to/maintain/identify-slow-queries.md [hunk 8]
```diff
@@ -39,13 +42,26 @@ Slow Query 基础信息：
...
+* `Commit_time`：表示事务提交时 commit 阶段的耗时。
```
Comment (zz-jason) [inline dev/how-to/maintain/identify-slow-queries.md]:
```suggestion
* `Commit_time`：表示事务两阶段提交中第二阶段（commit 阶段）的耗时。
```

File: dev/how-to/maintain/identify-slow-queries.md [hunk 9]
```diff
@@ -39,13 +42,26 @@ Slow Query 基础信息：
...
+* `Get_commit_ts_time`：表示事务提交时获取 commit tso 的耗时。
```
Comment (zz-jason) [inline dev/how-to/maintain/identify-slow-queries.md]:
```suggestion
* `Get_commit_ts_time`：表示事务两阶段提交中第二阶段（commit 阶段）获取 commit 时间戳的耗时。
```

File: dev/how-to/maintain/identify-slow-queries.md (hunk 4 above)
Comment (zz-jason) [inline dev/how-to/maintain/identify-slow-queries.md]:
```suggestion
* `Local_latch_wait_time`：表示事务两阶段提交中第二阶段（commit 阶段）发起前在 TiDB 侧等锁的耗时。
```

File: dev/how-to/maintain/identify-slow-queries.md [hunk 10]
```diff
@@ -39,13 +42,26 @@ Slow Query 基础信息：
...
+* `Local_latch_wait_time`：表示事务在提交前在本地等待 latch 的耗时。
+* `Write_keys`：表示事务提交时写 key 的数量。
```
Comment (zz-jason) [inline dev/how-to/maintain/identify-slow-queries.md]:
```suggestion
* `Write_keys`：表示该事务向 TiKV 的 Write CF 写入 Key 的数量。
```

File: dev/how-to/maintain/identify-slow-queries.md (hunk 3 above)
Comment (zz-jason) [inline dev/how-to/maintain/identify-slow-queries.md]:
```suggestion
* `Prewrite_region`：表示事务两阶段提交中第一阶段（prewrite 阶段）涉及的 TiKV Region 数量。注：每个 Region 会触发一次远程过程调用。
```

**Comment by crazycs520** [comment]
@zz-jason PTAL

**Comment by TomShawn** [comment]
@crazycs520 When the content is ready, please also update v3.1 as well, thanks~

**Comment by zimulala** [comment]
Please address the comments. @crazycs520

File: dev/how-to/maintain/identify-slow-queries.md [hunk 11]
```diff
@@ -39,13 +42,26 @@ Slow Query 基础信息：
...
+
+和事务执行相关的字段：
+
+* `Prewrite_time`：表示事务两阶段提交中第一阶段（prewrite 阶段）的耗时。
+* `Commit_time`：表示事务两阶段提交中第二阶段（commit 阶段）的耗时。
+* `Get_commit_ts_time`：表示事务两阶段提交中第二阶段（commit 阶段）获取 commit 时间戳的耗时。
+* `Local_latch_wait_time`：表示事务两阶段提交中第二阶段（commit 阶段）发起前在 TiDB 侧等锁的耗时。
+* `Write_keys`：表示该事务向 TiKV 的 Write CF 写入 Key 的数量。
+* `Write_size`：表示事务提交时写 key 或 value 的总大小。
+* `Prewrite_region`：表示事务两阶段提交中第一阶段（prewrite 阶段）涉及的 TiKV Region 数量。注：每个 Region 会触发一次远程过程调用。
```
Comment (TomShawn) [inline dev/how-to/maintain/identify-slow-queries.md]:
```suggestion
* `Prewrite_region`：表示事务两阶段提交中第一阶段（prewrite 阶段）涉及的 TiKV Region 数量。每个 Region 会触发一次远程过程调用。
```

File: v3.0/how-to/maintain/identify-slow-queries.md [hunk 12]
```diff
@@ -39,13 +42,26 @@ Slow Query 基础信息：
...
+
+和事务执行相关的字段：
+
+* `Prewrite_time`：表示事务两阶段提交中第一阶段（prewrite 阶段）的耗时。
+* `Commit_time`：表示事务两阶段提交中第二阶段（commit 阶段）的耗时。
+* `Get_commit_ts_time`：表示事务两阶段提交中第二阶段（commit 阶段）获取 commit 时间戳的耗时。
+* `Local_latch_wait_time`：表示事务两阶段提交中第二阶段（commit 阶段）发起前在 TiDB 侧等锁的耗时。
+* `Write_keys`：表示该事务向 TiKV 的 Write CF 写入 Key 的数量。
+* `Write_size`：表示事务提交时写 key 或 value 的总大小。
+* `Prewrite_region`：表示事务两阶段提交中第一阶段（prewrite 阶段）涉及的 TiKV Region 数量。注：每个 Region 会触发一次远程过程调用。
```
Comment (TomShawn) [inline v3.0/how-to/maintain/identify-slow-queries.md]:
```suggestion
* `Prewrite_region`：表示事务两阶段提交中第一阶段（prewrite 阶段）涉及的 TiKV Region 数量。每个 Region 会触发一次远程过程调用。
```

File: v3.1/how-to/maintain/identify-slow-queries.md [hunk 13]
```diff
@@ -38,13 +41,26 @@ Slow Query 基础信息：
...
+
+和事务执行相关的字段：
+
+* `Prewrite_time`：表示事务两阶段提交中第一阶段（prewrite 阶段）的耗时。
+* `Commit_time`：表示事务两阶段提交中第二阶段（commit 阶段）的耗时。
+* `Get_commit_ts_time`：表示事务两阶段提交中第二阶段（commit 阶段）获取 commit 时间戳的耗时。
+* `Local_latch_wait_time`：表示事务两阶段提交中第二阶段（commit 阶段）发起前在 TiDB 侧等锁的耗时。
+* `Write_keys`：表示该事务向 TiKV 的 Write CF 写入 Key 的数量。
+* `Write_size`：表示事务提交时写 key 或 value 的总大小。
+* `Prewrite_region`：表示事务两阶段提交中第一阶段（prewrite 阶段）涉及的 TiKV Region 数量。注：每个 Region 会触发一次远程过程调用。
```
Comment (TomShawn) [inline v3.1/how-to/maintain/identify-slow-queries.md]:
```suggestion
* `Prewrite_region`：表示事务两阶段提交中第一阶段（prewrite 阶段）涉及的 TiKV Region 数量。每个 Region 会触发一次远程过程调用。
```

**Comment by crazycs520** [comment]
@TomShawn PTAL

**Pull request merged**